components/core/Base/AntInputComponentBase.cs

```diff
@@ -94,7 +94,7 @@ public virtual TValue Value
         public string Size { get; set; } = AntSizeLDSType.Default;
 
         [Parameter]
-        public CultureInfo CultureInfo { get; set; } = CultureInfo.CurrentCulture;
+        public virtual CultureInfo CultureInfo { get; set; } = CultureInfo.CurrentCulture;
```

Comment:
I think the problem we have here is that `Locale` is trying to do the same thing as `CultureInfo` plus some `AntDesign` specific stuff. I agree it is still needed, but possibly could be hidden behind `CultureInfo` and not really exposed to the user. Maybe as an option, through DI, for advanced scenarios. This way `CultureInfo` could be used as the only definition of culture/locale. Appropriate json would be retrieved based on `CultureInfo` to cover the `AntDesign` specific requirements. This way we could skip the double-data entry. Or allow json to "override" `CultureInfo` if necessary. I think there was an idea to "redo" the whole localization logic, but no one decided to tackle this problem, and this is indeed a problem as it will affect the whole library. @ElderJames what do you think?